src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,16 @@
+// FOR STARS
+@mixin star($size) {
+  width: $star-size;
+  height: $star-size;
+}
+
+// FOR LOGO
+@mixin squareLogo($size) {
+  width: $logo-size;
+  height: $logo-size;
+}
```

Comment:
But it's the same mixins 🙂  Create one mixin for square and use it for logo and stars